pages/reviewer/PKPReviewerHandler.php

```diff
@@ -205,13 +207,13 @@ public function getReviewForm(
     ): ReviewerReviewForm {
         switch ($step) {
             case 1:
-                return new \PKP\submission\reviewer\form\PKPReviewerReviewStep1Form($request, $reviewSubmission, $reviewAssignment);
+                return new PKPReviewerReviewStep1Form($request, $reviewSubmission, $reviewAssignment);
             case 2:
-                return new \PKP\submission\reviewer\form\PKPReviewerReviewStep2Form($request, $reviewSubmission, $reviewAssignment);
+                return new PKPReviewerReviewStep2Form($request, $reviewSubmission, $reviewAssignment);
             case 3:
-                return new \PKP\submission\reviewer\form\PKPReviewerReviewStep3Form($request, $reviewSubmission, $reviewAssignment);
+                return new PKPReviewerReviewStep3Form($request, $reviewSubmission, $reviewAssignment);
             default:
-                return null;
+                return throw new Exception("Invalid review form step {$step}");
```

Comment:
It does appear to work, but is `return throw` an intentional choice rather than just `throw`? The `return throw` syntax is not a common formulation in PHP.

Reply:
Typo :)